populateStds.js

```diff
@@ -93,18 +108,6 @@ filenames.forEach((filename) => {
     }
 })
 
-function readPrefixLines(fn) {
-    var lines = []
-    if (fs.existsSync(fn)) {
-        lines = fs.readFileSync(fn, 'utf8').split('\n')
-        var tableIdx = lines.findIndex((line) => line.trim().startsWith('|'))
-        if (tableIdx >= 0) {
-            lines.splice(tableIdx)
-        }
-    } else console.log(`WARNING: file ${fn} not found`)
-    return lines
-}
```

Comment:
When developing on my local computer I noticed that this function is not beneficial, so I excluded it.
The problem was, that the header was not updated properly (the included react code was not executed anymore), therefore I always needed to delete the overview.mdx manually, when I made changes.
I also was not really able to figure out what the purpose is.
If anyone thinks that this function is needed and should stay, please explain why.

Reply:
I think it's no longer necessary. I can't tell what the problem was in your case. You could supply the non-working `overview.mdx` for further inspection.

Reply:
Since it is ok from your side to have it excluded, we do not need to track down the problem imo. I already checked it, but was not able to identify any obvious differences. I think there would be the need to check for certain characters and how they are used.

Reply:
Agreed